internal/util/util_test.go

```diff
@@ -0,0 +1,196 @@
+package util
+
+import (
+	"os"
+	"reflect"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func Test_HomeDir(t *testing.T) {
+	tests := []struct {
+		name string
+		env  string
+		dir  string
+		want string
+	}{
+		{
+			name: "test linux/unix home directory",
+			env:  "HOME",
+			dir:  "/home/test",
+			want: "/home/test",
+		},
+		{
+			name: "test windows home directory",
+			env:  "USERPROFILE",
+			dir:  "/users/test",
```

Comment:
I'd suggest trying a `Users` test here (capital U), which is the standard on Windows. may not make a difference, but it's more accurate to be safe :)

Reply:
Thank you @CpuID. Good catch, I have change the windows test path to `C:\Users\test` should be more accurate